.github/workflows/windows.yml

```diff
@@ -0,0 +1,56 @@
+name: windows
+
+on:
+  push:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'

```

Comment:
add:
```
- '!.github/workflows/EnterDevShell.ps1'
```

That's `paths-ignore`, idea is that if there is change only to other .yml workflows, avoid running this one. But you want to run it if there is a change to EnterDevShell.ps1.